install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -458,6 +487,16 @@ spec:
           type: DirectoryOrCreate
         name: bpf-maps
 {{- end }}
+      # To mount cgroup2 filesystem on the host
+      - hostPath:
+          path: /proc/1/ns
+          type: DirectoryOrCreate
+        name: host-proc-ns
+      # To keep state between restarts / upgrades for cgroup2 filesystem
+      - hostPath:
+          path: {{ .Values.cgroup.hostRoot}}
+          type: DirectoryOrCreate
+        name: cilium-cgroup
```

Comment:
Is this really necessary? I thought the cgroup2 state would be kept even if it was not mounted in the host. I might be wrong but that was my impression.

Reply:
>cgroup2 state would be kept even if it was not mounted in the host

Not sure I understand. Where would the state be kept?

Reply:
oh this is the hostRoot ok, nevermind.